cloudflared/config.yaml

```diff
@@ -21,10 +21,20 @@ map:
 options:
   external_hostname: null
   tunnel_name: "homeassistant"
+  strict_ssl: false
```

Comment:
As mentioned above, I would reduce all these options to just a normal optional option: 

```
schema:
   ...
   generate_cert:bool?
   ...
```